src/Google.Bigquery.V2/Preconditions.cs

```diff
@@ -0,0 +1,168 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Should this be called something like "BigqueryPreconditions.cs"? In a similar manner to GaxPreconditions?